src/test/java/com/qubole/utility/RedisCacheTest.java

```diff
@@ -67,7 +64,7 @@ public void testGetFromSource() throws Exception {
     JedisPool jedisPool = mock(JedisPool.class);
     Jedis jedis = mock(Jedis.class);
     when(jedis.get(anyString().getBytes())).thenThrow(new JedisConnectionException("Get call failed."));
-    when(jedisPool.getResource()).thenReturn(jedis);
```

Comment:
Hmm - this doesnt affect positive tests ?

Reply:
Connection exception from 'jedis.get' and 'jedisPool.getResource' end up getting caught and handled the same way. So in either case we fallback to loading the data from source.

Reply:
Test setup for each test is different